tests/server/tso/manager_test.go

```diff
@@ -133,8 +136,23 @@ func (s *testPrioritySuite) TestAllocatorPriority(c *C) {
 	// pd2: dc-2 allocator leader
 	// pd3: dc-3 allocator leader
 
-	// Because the default priority checking period is 1 minute,
-	// so we sleep longer here.
+	// To speed up the test, we force to do the check
+	for _, server := range cluster.GetServers() {
+		server.GetTSOAllocatorManager().ClusterDCLocationChecker()
+	}
+	// Wait for each DC's Local TSO Allocator leader
+	for _, dcLocation := range dcLocationConfig {
+		testutil.WaitUntil(c, func(c *C) bool {
+			leaderName := cluster.WaitAllocatorLeader(dcLocation)
+			return len(leaderName) > 0
+		})
+	}
+	// Same as above
+	for _, server := range cluster.GetServers() {
+		server.GetTSOAllocatorManager().PriorityChecker()
+	}
+	// Because the leader changing may take quite a long period,
+	// so we sleep longer here to wait.
 	time.Sleep(waitAllocatorPriorityCheckInterval)
```

Comment:
“sleep longer” but 2 minutes to 3 minutes?

Reply:
“longer” is compared with the leader changing duration.